Controllers/InterventionViewController.swift

```diff
@@ -356,18 +311,14 @@ class InterventionViewController: UIViewController, UITableViewDelegate, UITable
   private func loadSampleInterventions() {
```

Comment:
At some point we'll need to move this somewhere else. (It isn't the controller's responsibility to make up some test data.)

Reply:
We're deleting this method in another branch (graphQL related branch I think). Since we fetch interventions, we no longer need samples